XMonad/Actions/CycleWS.hs

```diff
@@ -298,9 +298,9 @@ hiddenWS = WSIs $ do
 anyWS :: WSType
 anyWS = WSIs . return $ const True
 
--- | cycle through workspaces that are not in the given list. This could be used
---   ,for example, to skip the workspace used for skipping the workspace reserved
---   for "XMonad.Util.NamedScratchpad":
+-- | Cycle through workspaces that are not in the given list. This could be used
+--   , for example, be used for skipping the workspace reserved for
+--   "XMonad.Util.NamedScratchpad":
```

Comment:
I realized that this still doesn't sound quite right :)

```suggestion
-- | Cycle through workspaces that are not in the given list. This could, for
--   example, be used for skipping the workspace reserved for
--   "XMonad.Util.NamedScratchpad":
```